libs/ui-components/src/components/form/validations.ts

```
@@ -695,3 +695,30 @@ export const deviceApprovalValidationSchema = (t: TFunction, conf: { isSingleDev
         ),
     labels: validLabelsSchema(t, forbiddenDeviceLabels),
   });
+
+export const createMassResumeValidationSchema = (t: TFunction) =>
+  Yup.object().shape({
+    mode: Yup.string().oneOf(['fleet', 'labels', 'all']).required(),
+    fleetId: Yup.string().when('mode', {
+      is: 'fleet',
+      then: (schema) => schema.required(t('Fleet selection is required')),
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)